build/versionize-docs.sh

```diff
@@ -72,4 +72,5 @@ done
 $SED -ri -e "s|(releases.k8s.io)/[^/]+|\1/${NEW_VERSION}|" pkg/api/v[0-9]*/types.go
 
 ${KUBE_ROOT}/hack/update-generated-docs.sh
-${KUBE_ROOT}/hack/update-swagger-spec.sh
+${KUBE_ROOT}/hack/update-generated-swagger-docs.sh
+
```

Comment:
Could you also add hack/gen-swagger-doc/run-gen-swagger-docs.sh at last? That script will generate the docs/api-reference/definitions.html and operations.html. Thanks.